.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly add the `node_modules` directory to the `.gitignore` file.
 
It is always a good idea to add the node_modules directory to the `.gitignore` file in all your projects as it is part of the standard practice. 
This way you can avoid putting possibly hundreds of MB of dependencies in your repository. 😄